commons/jdk11/src/main/java/org/infinispan/commons/jdkspecific/ClasspathURLStreamHandler.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.commons.jdkspecific;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLStreamHandler;
+import java.net.spi.URLStreamHandlerProvider;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 12.1
+ **/
+@MetaInfServices
+public class ClasspathURLStreamHandler extends URLStreamHandlerProvider {
```

Comment:
How does this implementation differ from the pre-java 11 implementation?

It would be nice to have a description in the javadoc.